Formula/libgetdata.rb

```diff
@@ -12,14 +12,14 @@ class Libgetdata < Formula
   end
 
   option "with-fortran", "Build Fortran 77 bindings"
-  option "with-perl", "Build Perl binding"
```

Comment:
@UniqMartin I sympathize with what you're saying, but it seems like the meaning of a perl *option* is obvious, so if anything your concern would suggest the default wording of :perl => :optional/:recommended (or, for that matter, :python => :optional/:recommended) should be changed if it's not sufficiently clear what building with perl support or building with python support means without further customized option descriptions.